contrib/ci/Jenkinsfile.osx

```
@@ -34,15 +34,18 @@ pipeline
       agent none
       steps
       {
+        githubNotify context: 'Jenkins: OSX', description: 'initializing...',  status: 'PENDING'
         // kill older builds in this PR:
         script { killold.killOldBuilds() }
       }
     }
 
     stage("main")
     {
-      agent {
-        node {
+      agent
+      {
+        node
+        {
           label 'osx'
```

```suggestion
          label 'jenkins: osx'
```
Just a shot in the dark. Maybe this label needs to be adjusted as well?

---

No. The machine label is something different